server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -67,14 +66,6 @@ public Form<AddApplicantToTrustedIntermediaryGroupForm> addNewClient(
     return form;
   }
 
-  private Form<AddApplicantToTrustedIntermediaryGroupForm> validateEmailAddress(
-      Form<AddApplicantToTrustedIntermediaryGroupForm> form) {
```

Comment:
I don't think we should get rid of it. If the email address is going to be optional, we still want to make sure it is unique.